src/protocol/server.c

```diff
@@ -125,11 +108,15 @@ protocolServerError(ProtocolServer *this, int code, const String *message, const
 
 /**********************************************************************************************************************************/
 void
-protocolServerProcess(ProtocolServer *this, const VariantList *retryInterval)
+protocolServerProcess(
+    ProtocolServer *this, const VariantList *retryInterval, const ProtocolServerHandler *const handlerList,
+    const unsigned int handlerListSize)
 {
     FUNCTION_LOG_BEGIN(logLevelDebug);
         FUNCTION_LOG_PARAM(PROTOCOL_SERVER, this);
         FUNCTION_LOG_PARAM(VARIANT_LIST, retryInterval);
+        FUNCTION_LOG_PARAM_P(VOID, handlerList);
+        FUNCTION_LOG_PARAM(UINT, handlerListSize);
```

Comment:
Yes, definitely need asserts here, added in 127aaf348a57. I can't find `protocolServerProcess(server, NULL)` -- you may be looking at the wrong branch.

Reply:
Line 182 of module/performance/storageTest.c has protocolServerProcess(server, NULL); - this is what is confusing me.

Reply:
Not sure how I missed that -- must have been looking in the wrong branch! Anyway, it wasn't causing problems because that test was not being run in CI. Looks like an oversight when we moved to Github Actions. Not too big a deal since it does not provide coverage but nice if it works when we need it for performance testing. So, fixed that separately in integration 0eb5b22ffa02b. The test was fixed in 96b0afcfa0df3dfa1.